### PR TITLE
Feat: Add rustywind formatter

### PR DESCRIPTION
**Pull request by emilford**
This adds rustywind, a CLI for organizing Tailwind CSS classes.

### CODE REVIEW

File: lua/conform/formatters/rustywind.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/avencera/rustywind",
+    description = "Inspired by Ryan Heybourn's headwind vscode plugin. This is a CLI tool that will look through your project and sort all Tailwind CSS classes. It will also delete any duplicate classes it finds.",
```
Comment (stevearc) [inline lua/conform/formatters/rustywind.lua:5]: Could you change this description to be `A tool for formatting Tailwind CSS classes.`? This is the string the will appear in the list of formatters.

**Review by stevearc — CHANGES_REQUESTED** [review]
Thanks for the PR! One documentation change requested

File: lua/conform/formatters/rustywind.lua (hunk 1 above)
Comment (emilford) [reply lua/conform/formatters/rustywind.lua]: Sure thing. Updated in 2da69125ec016799d91e7bb3fcacd029a379b4cd.

**Comment by stevearc** [comment]
Thanks!

**Pull request merged**